esti/commit_test.go

```diff
@@ -12,33 +12,25 @@ import (
 )
 
 func TestCommitSingle(t *testing.T) {
-	for _, direct := range testDirectDataAccess {
-		name := "indirect"
-		if direct {
-			name = "direct"
-		}
-		t.Run(name, func(t *testing.T) {
-			ctx, _, repo := setupTest(t)
-			defer tearDownTest(repo)
-
-			const objPath = "1.txt"
-			_, objContent := uploadFileRandomData(ctx, t, repo, mainBranch, objPath, direct)
-			commitResp, err := client.CommitWithResponse(ctx, repo, mainBranch, &apigen.CommitParams{}, apigen.CommitJSONRequestBody{
-				Message: "singleCommit",
-			})
-			require.NoError(t, err, "failed to commit changes")
-			require.NoErrorf(t, verifyResponse(commitResp.HTTPResponse, commitResp.Body),
-				"failed to commit changes repo %s branch %s", repo, mainBranch)
-
-			getObjResp, err := client.GetObjectWithResponse(ctx, repo, mainBranch, &apigen.GetObjectParams{Path: objPath})
-			require.NoError(t, err, "failed to get object")
-			require.NoErrorf(t, verifyResponse(getObjResp.HTTPResponse, getObjResp.Body),
-				"failed to get object repo %s branch %s path %s", repo, mainBranch, objPath)
-
-			body := string(getObjResp.Body)
-			require.Equalf(t, objContent, body, "path: %s, expected: %s, actual:%s", objPath, objContent, body)
-		})
-	}
+	ctx, _, repo := setupTest(t)
```

Comment:
Unrelated question to this PR- how do we check that everything works with pre-signed mode?

Reply:
presign_test.go

Reply:
I think that a better way to test it is to run all the tests in another suit with the pre-signed flag on so we don't have to remember to add a new test for every command/feature we add but again, it doesn't related to this PR